src/client/app/containers/CompareChartContainer.js

```diff
@@ -0,0 +1,123 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import _ from 'lodash';
+import { Bar } from 'react-chartjs-2';
+import moment from 'moment';
+import { connect } from 'react-redux';
+import datalabels from 'chartjs-plugin-datalabels';
+import GraphColors from '../utils/GraphColors';
+
+
+/**
+ * @param {State} state
+ */
+function mapStateToProps(state) {
+	const timeInterval = state.graph.compareTimeInterval;
+	const barDuration = state.graph.compareDuration;
+	const data = { datasets: [] };
+	const labelsSet = new Set();
```

Comment:
labelsSet does not need to be a set, just a list. Also, change the variable name to `labels`.